mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -379,25 +379,43 @@ class RowMatrix @Since("1.0.0") (
    * Note that this cannot be computed on matrices with more than 65535 columns.
    *
    * @param k number of top principal components.
-   * @return a matrix of size n-by-k, whose columns are principal components
+   * @return a matrix of size n-by-k, whose columns are principal components, and
+   * a vector of values which indicate how much variance each principal component
```

Comment:
It is not very clear from the doc whether we return the absolute variance explained or the proportions. How about `a vector of proportions of variance explained by each principal component` and change the method name to `computePrincipalComponentsAndExplainedVariance`?